### PR TITLE
Tidy up the change log

### PR DESCRIPTION
**Pull request by chrisseaton**
We don't know which commit finally gets used for the release, so let's make the change log ready to use at any time.

### CODE REVIEW

**Comment by pitr-ch** [comment]
Maybe, add support of --native option in env.var. is worth adding.

**Comment by chrisseaton** [comment]
Yes please.

**Comment by chrisseaton** [comment]
This should be unstuck by #1150.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+# 0.32, end of February 2018
 
 New features:
 
-* ...
-
-Bug fixes:
-
-* ...
+* A new embedded configuration is used when TruffleRuby is used from another
```
Comment (eregon) [inline CHANGELOG.md]: It also enforces a single thread.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,7 +14,8 @@ Performance:
 Internal changes:
 
 * The launcher is now part of the TruffleRuby repository, rather than part of
-the GraalVM repository.
+the GraalVM repository and it accepts `--native` and similar options in 
+`TRUFFLERUBYOPT` environment variable.
```
Comment (eregon) [inline CHANGELOG.md]: The `--native` in TRUFFLERUBYOPT is not an internal change but a new feature or bug fix.
"in *the* TRUFFLERUBYOPT` environment variable"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
 New features:
 
-* ...
-
-Bug fixes:
-
-* ...
+* A new embedded configuration is used when TruffleRuby is used from another
+language or application. This disables features like signals which may
+conflict with the embedding application such as threads, and enables features
```
Comment (eregon) [inline CHANGELOG.md]: like ... with ...
This disables features like signals and threads which may conflict with the embedding application, and ...

Comment (chrisseaton) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (chrisseaton) [reply CHANGELOG.md]: Fixed.

**Pull request merged**